website/source/docs/commands/force-leave.html.markdown.erb

```diff
@@ -10,20 +10,24 @@ description: |-
 
 Command: `consul force-leave`
 
-The `force-leave` command forces a member of a Consul cluster to enter the
-"left" state. If the member is still actually alive, it will
-eventually rejoin the cluster. The true purpose of this method is to force
-remove "failed" nodes.
+The `force-leave` command forces a departed member of a Consul cluster to enter the
```

Comment:
Instead of departed, do you feel something like "unhealthy" may be more fitting? I feel like this word choice caters more to the idea of nodes in left state and not failed.

Reply:
Going to remove that world altogether.